docs/reference/configuration/queue.rst

```diff
@@ -355,6 +364,23 @@ The following is a list of all queue-specific configuration options:
 
     QUEUE_OPTION TORQUE QUEUE_QUERY_TIMEOUT 254
 
+.. _torque_project_code:
+.. topic:: PROJECT_CODE
+
```

Comment:
This is not your doing, but the `PROJECT_CODE` topic does not have an initial description like other topics have.
Does it make sense to add something simplistic to indicate what `PROJECT_CODE` implies, rather than immediately refer to an external link to man pages for qsub/bsub etc?
```
String identifier used to map hardware resource usage to project and/or account
```

Reply:
Sounds like a good thing to add.